docs/api/flink/Function.md

```diff
@@ -1787,3 +1787,26 @@ SELECT ST_ZMin(ST_GeomFromText('LINESTRING(1 3 4, 5 6 7)'))
 ```
 
 Output: `4.0`
+
+
+##  ST_CoordDim
```

Comment:
1. Doc are sorted alphabetically.
2. Please include since `v1.5.0`
3. Do not say `Spark SQL example` since this doc is for Flink.